apps/financial-aid/web-osk/src/components/HasApplied/hasApplied.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState, useContext } from 'react'
+import {
+  Text,
+  Icon,
+  Box,
+  Checkbox,
+  GridContainer,
+} from '@island.is/island-ui/core'
+
+import {
+  FormContentContainer,
+  FormFooter,
+  FormLayout,
+  LogoHfj,
+} from '@island.is/financial-aid-web/osk/src/components'
+import * as styles from './hasApplied.treat'
+import { useRouter } from 'next/router'
+
+import { UserContext } from '@island.is/financial-aid-web/osk/src/components/UserProvider/UserProvider'
```

Comment:
There are a bunch of imports here that are not used in this file, we should remove them